README.md

```diff
@@ -112,7 +112,7 @@ The supported commands are divided into groups according to their target, at thi
     export default migration;
     ```
 
-* `migration run` - Runs a migration script specified by file name (option `--name <file name>`), or runs multiple migration scripts in the order specified in the migration files (option `--all`).
+* `migration run` - Runs a migration script specified by file name (option `--name <file name>`), or runs multiple migration scripts in the order specified in the migration files (options `--all` or `--range`).

```

Comment:
We might want to mention that the order does not need to be incremental by 1 (can have gaps)